test/AlexaSkill.spec.js

```diff
@@ -11,6 +11,11 @@ const simple = require('simple-mock');
 const _ = require('lodash');
 
 describe('AlexaSkill', () => {
+  it('Skill should able to run without any port as a parameter', () => {
```

Comment:
this is not really testing anything, we should either remove the test or rewrite it so it actually makes an assert